packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx

```diff
@@ -60,6 +60,11 @@ export type CallContentProps = Pick<CallControlProps, 'onHangupCallHandler'> &
      * This switches the participant's layout between the grid and the spotlight mode.
      */
     layout?: 'grid' | 'spotlight';
+    /**
+     * Check if phone is in landscape mode.
```

Comment:
```suggestion
     * Check if device is in landscape mode.
```

could be a tablet also